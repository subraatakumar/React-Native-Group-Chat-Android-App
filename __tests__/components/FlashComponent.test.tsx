import React from 'react';
import renderer from 'react-test-renderer';
import FlashScreen from '../../src/components/FlashComponent';

test('Flash Component renders correctly', () => {
  const flashComponent = renderer.create(<FlashScreen />);
  expect(flashComponent).toMatchSnapshot();
});
