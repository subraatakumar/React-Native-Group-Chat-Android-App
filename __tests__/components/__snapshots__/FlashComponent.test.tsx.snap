// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Flash Component renders correctly 1`] = `
<Modal
  hardwareAccelerated={false}
  visible={Object {}}
/>
`;
